src/instructlab/sdg/filterblock.py

```diff
@@ -37,6 +37,13 @@ def generate(self, samples) -> Dataset:
                 num_proc=self.num_procs,
             )
 
+        if isinstance(self.value, list):
```

Comment:
Since we all found this stuff some confusing, simplifying like this might help:

```
class FilterByValueBlock(Block):
    def __init__(...):
        self.value = filter_value
        if not isinstance(self.value, list):
            self.value = [self.value]
```

and then just have one filter code path

Reply:
Good suggestion. That sounds nicer / smaller / clearer

Reply:
applied this suggestion, thanks!